package/MDAnalysis/topology/TOPParser.py

```diff
@@ -140,13 +141,20 @@ class TOPParser(TopologyReaderBase):
     - Bonds
     - Angles
     - Dihedrals (inc. impropers)
+    - ChainIDs (from %RESIDUE_CHAINID)
+    - Segids (from %RESIDUE_CHAINID)
 
     The format is defined in `PARM parameter/topology file
     specification`_.  The reader tries to detect if it is a newer
     (AMBER 12?) file format by looking for the flag "ATOMIC_NUMBER".
 
     .. _`PARM parameter/topology file specification`:
-       http://ambermd.org/formats.html#topology
+       https://ambermd.org/FileFormats.php#topo.cntrl
+
+    Additionally, the RESIDUE_CHAINID non-standard flag is supported. This
```

Comment:
Add versionchanged entry?

Reply:
sorry, I didn't get this.

Reply:
On lines 165 to 172 we have versionchanged entries that document changes we make to this reader. If possible could you add an entry that documents these new changes?

Reply:
Done.